nertokenizer.py

```diff
@@ -301,3 +313,9 @@ def add_to_state(slist: List[str], entity: Entity) -> None:
     # print("\nLast names:\n{0}".format("\n".join("{0}: {1}".format(k, v) for k, v in lastnames.items())))
 
     assert not tq
+
+
+if __name__ == "__main__":
```

Comment:
Delete debug (also delete `import pprint` at top of file)

Reply:
Done